docs/level-20.md

```diff
@@ -32,10 +32,6 @@ title: Level 20 - Ignition
 
 <br />
 
-## Special Moves
```

Comment:
don't remove this

Reply:
I don't understand what value this has - each of the 'Special Moves' listed are just different types of Double Ignitions - seems redundant, no?

Reply:
every section follows this format, so you should keep it for consistency